src/main/java/org/embulk/input/jira/util/JiraUtil.java

```diff
@@ -61,21 +66,22 @@ public static void validateTaskConfig(final PluginTask task)
         if (isNullOrEmpty(uri)) {
             throw new ConfigException("JIRA API endpoint could not be empty");
         }
-        HttpURLConnection connection = null;
-        try {
-            URL u = new URL(uri);
-            connection = (HttpURLConnection) u.openConnection();
-            connection.setRequestMethod("GET");
-            connection.getResponseCode();
+        try (CloseableHttpClient client = HttpClientBuilder.create()
+                                            .setDefaultRequestConfig(RequestConfig.custom()
+                                                                                .setConnectTimeout(HTTP_TIMEOUT)
+                                                                                .setConnectionRequestTimeout(HTTP_TIMEOUT)
+                                                                                .setSocketTimeout(HTTP_TIMEOUT)
+                                                                                .setCookieSpec(CookieSpecs.STANDARD)
```

Comment:
Same as on `JiraClient`:
> Question: what is the relation of "cookie spec" here with the current issue?

Reply:
This is due to legacy of HTTP Client of Apache. If we do not set it, we will get a logging warning in header 
```
Invalid 'expires' attribute
```